.devcontainer/README.md

```diff
@@ -0,0 +1,20 @@
+# Purpose of the devcontainer
+
+This devcontainer uses a `Dockerfile` and is meant to allow reproducible R CMD checks of XLConnect, as well as reverse dependency checks before new XLConnect releases.
```

Comment:
The purpose is general development of xlconnect using VS code, with minimal dependency setup.
It would be good to add the link https://microsoft.github.io/code-with-engineering-playbook/developer-experience/devcontainers/

Reply:
I updated the sentence and added a link to that help page.